app/templates/components/public/ticket-list.hbs

```diff
@@ -153,13 +153,14 @@
   <div class="ui grid">
     <div class="ui row stackable grid">
       <div class="ui less padding thirteen wide computer twelve wide tablet column right aligned floated payment icons">
-        <i class="colored american express link icon"></i>
-        <i class="colored credit card alternative link icon"></i>
-        <i class="colored diners club link icon"></i>
-        <i class="colored discover link icon"></i>
-        <i class="colored mastercard link icon"></i>
-        <i class="colored paypal card link icon"></i>
-        <i class="colored visa link icon"></i>
+        <i class="blue cc amex link icon"></i>
```

Comment:
Also, the icons you used and icons which are being used a whole lot different.

Reply:
I take them from https://semantic-ui.com/elements/icon.html#brands, there were no cards in https://semantic-ui.com/elements/icon.html#payments--shopping .